src/profile-logic/marker-data.js

```diff
@@ -162,6 +163,27 @@ export function getSearchFilteredMarkerIndexes(
   return newMarkers;
 }
 
+export function getTabFilteredMarkerIndexes(
```

Comment:
We should probably audit some special cases here. I'd probably want to retain "global" things, like the responsiveness markers.

Reply:
I think maybe we should file a follow-up for this point. Is there already one on file?

Reply:
Re-resolving.